resources/public/api/conf/1.0/schemas/create-organisation-response.json

```diff
@@ -149,6 +149,10 @@
       "type": "string",
       "description": "Company Reference Number"
     },
+    "personPresentingTheGoods": {
```

Comment:
This should not be part of this PR - it fits better with https://github.com/hmrc/api-platform-test-user/pull/177

Reply:
updated as per review comments